package.json

```
@@ -12,6 +12,9 @@
 	},
 	"homepage": "https://github.com/AAU-SW7-PSAAB/energy-label-extension#readme",
 	"type": "module",
+	"engines": {
+		"node": ">=22.0.0"
```

```suggestion
		"node": ">=22.1.0"
```
Ummm akshtually it was added in v22.1.0

---

🤓